airflow/dag_processing/processor.py

```diff
@@ -234,8 +243,16 @@ def _handle_request(self, msg: ToParent, log: FilteringBoundLogger) -> None:  #
         if isinstance(msg, DagFileParsingResult):
             self.parsing_result = msg
             return
-        # GetVariable etc -- parsing a dag can run top level code that asks for an Airflow Variable
-        super()._handle_request(msg, log)
```

Comment:
We wont really need this, because for cases of variables, connecitons, we will have to interact with the DB model directly. If we go to `super(). _handle_request`, it brings the SDK API client into picture, which shouldn't be needed for DAG level stuff